lib/js_render/console_replay.js

```diff
@@ -0,0 +1,19 @@
+(function(history) {
+  var result = '';
```

Comment:
can we built up an array of strings and then join then when we return instead? (I think this will be more performant)

Reply:
Doing some cursory benching shows concats performing better than array joins: http://jsben.ch/v6WAP. This is likely browser specific, though. What do you think?

Reply:
Woah that's crazy. Firefox is the opposite though: http://jsben.ch/0UILM. I guess Chrome made some optimizations